cluster-api/src/main/java/io/aiven/klaw/clusterapi/services/KafkaConnectService.java

```diff
@@ -30,6 +34,12 @@ public class KafkaConnectService {
       new ParameterizedTypeReference<>() {};
   private static final ParameterizedTypeReference<Map<String, Object>>
       GET_CONNECTOR_DETAILS_TYPEREF = new ParameterizedTypeReference<>() {};
+  public static final String UNABLE_TO_CREATE_CONNECTOR_ON_CLUSTER =
```

Comment:
Would it be better with a err code and terse

Reply:
Can you give an example?

Reply:
We can move all these error error descriptions to another class like KlawErrorMessages ?
CONNECT_ERR_101